rgbd/src/client_ros.cpp

```diff
@@ -34,12 +30,9 @@ bool ClientROS::initialize(const std::string& rgb_image_topic, const std::string
 {
     nh_.setCallbackQueue(&cb_queue_);
 
-    sub_cam_info_ = nh_.subscribe(cam_info_topic, 1, &ClientROS::camInfoCallback, this);
+    if (!ClientROSBase::initialize(rgb_image_topic, depth_image_topic, cam_info_topic))
```

Comment:
Why `ClientROSBase`? `ClientROS` is already inheriting it so `initialize` should already be available to the class

Reply:
But this is in Client Ros:: initialize. We need to call the function of the base class.

Reply:
I didn't see the function which was calling it. You are right :)